<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Font2lcd</string>
  </property>
  <property name="windowIcon">
   <iconset resource="my.qrc">
    <normaloff>:/icons/img/font2lcd.ico</normaloff>:/icons/img/font2lcd.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Paint" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>363</width>
      <height>363</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="push_font">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>font</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>482</width>
      <height>482</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>16</number>
    </property>
    <property name="columnCount">
     <number>16</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="push_pen">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>60</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>font color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_brush">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>background color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_xy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="push_save">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>140</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_progress">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>751</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_file">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>621</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>661</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="sliderTop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderBottom">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>470</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>upper limit</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lower limit</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_font">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>table</zorder>
   <zorder>widget</zorder>
   <zorder>push_font</zorder>
   <zorder>push_pen</zorder>
   <zorder>push_brush</zorder>
   <zorder>label_xy</zorder>
   <zorder>push_save</zorder>
   <zorder>frame_progress</zorder>
   <zorder>sliderTop</zorder>
   <zorder>sliderBottom</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_font</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionChange"/>
   <addaction name="actionPen"/>
   <addaction name="actionBrush"/>
   <addaction name="actionConfigure"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="my.qrc">
     <normaloff>:/icons/img/disk.png</normaloff>:/icons/img/disk.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="icon">
    <iconset resource="my.qrc">
     <normaloff>:/icons/img/font_32.png</normaloff>:/icons/img/font_32.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Change</string>
   </property>
   <property name="toolTip">
    <string>Change Fonts</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="my.qrc">
     <normaloff>:/icons/img/setting_tools.png</normaloff>:/icons/img/setting_tools.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="my.qrc">
     <normaloff>:/icons/img/Shutdown.ico</normaloff>:/icons/img/Shutdown.ico</iconset>
   </property>
   <property name="text">
    <string notr="true">Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="icon">
    <iconset resource="my.qrc">
     <normaloff>:/icons/img/brushes.png</normaloff>:/icons/img/brushes.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Pen</string>
   </property>
   <property name="toolTip">
    <string>Change color pen</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="icon">
    <iconset resource="my.qrc">
     <normaloff>:/icons/img/document_background.png</normaloff>:/icons/img/document_background.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Brush</string>
   </property>
   <property name="toolTip">
    <string>Change color brush</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Paint</class>
   <extends>QWidget</extends>
   <header location="global">paint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="my.qrc"/>
 </resources>
 <connections/>
</ui>
